Gemfile.lock

```diff
@@ -645,4 +645,4 @@ RUBY VERSION
    ruby 2.6.5p114
 
 BUNDLED WITH
-   1.17.3
+   2.2.20
```

Comment:
**question:** I'm having trouble reconciling this with the title/description of the PR - which one is correct?

Reply:
I think the file change is correct, I don't see a 2.20.x release of bundler

Reply:
Makes sense, I didn't have time to check. Thanks!